### DISCUSSION TITLE
Adding Clip_at to render for indoor scenarios

### DISCUSSION THREAD

**Discussion by fsvsf** [NVlabs/sionna#398] [Q&A]
Hello,

I am currently working with Sionna RT coverage maps in indoor scenarios and I am having trouble to check the coverage map with scene.render to get high quality images of the coverage map. I know that scene.preview has a clip_at function added in the latest version. Would it be possible to add the clip_at function to the scene.render and scene.render_to_file?

Thank you in advance

**Comment by merlinND** [comment]
Hello @fsvsf,

While `clip_at` was relatively easy to implement in the `scene.preview()` renderer, which uses rasterization, unfortunately there is no equivalent in the physically  based renderer used by `scene.render()`.

I think that the easiest way to get a similar effect would be to assign a null BSDF to the objects you do not want to see in the rendering (e.g. roofs) in Blender, and exporting your scene again. Would that work for  you?
https://docs.blender.org/manual/en/latest/render/shader_nodes/shader/transparent.html

PS: you could use e.g. Blender's "select similar normal" option to efficiently select all roofs in order to assign the transparent BSDF to them.
